app/controllers/concerns/hyku_addons/users_controller_behavior.rb

```diff
@@ -12,7 +12,15 @@ def index
 
     def show
       @user = User.find_by(email: params[:email], display_profile: true)
-      render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } if @user.blank?
+      return render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } if @user.blank?
```

Comment:
Can you also make sure you're using double quotes, they are both being used through the method.